start:
  - 0
  - 0
  - 0
  - 0
  - 0

goal:
  - 2
  - 0
  - 0
  - 0
  - 3.14159

midpoints1:
  - [-1.723472, 0.573341, 0, 0, 0]
  - [0.68788, 0.5907, 0, 0, 0]
  - [0.67939, 1.764342, 0, 0, 1.5707]
  - [1.31922, 1.609342, 0, 0, -0.754729]

midpoints2:
  - [-1.08773, -1.755078, 0, 0, 0]
  - [0.595516, -1.73132, 0, 0, 1.5707]
  - [0.571807, -0.8312, 0, 0, 0.481632]

midpoints3:
  - [-0.5709, -0.538144, 0, 0, 0]
  - [-0.5709, 0.4965, 0, 0, 1.5707]
  - [0.06651, 0.4917, 0, 0, 0]
  - [0.63024, -0.59645, 0, 0, -1.5707]
